### DISCUSSION TITLE
"python3.6.10 supported" on the README.md?

### DISCUSSION THREAD

**Discussion by CyrilLeMat** [blue-yonder/tsfresh#978] [General]
python3.6 is not supported anymore. This should be replaced by more recent python versions (3.7, 3.8, 3.9)

**Comment by nils-braun** [comment]
Totally right @CyrilLeMat ! I have removed the badge - it was nice to have such a badge while doing the switch from python 2 to 3 (long long ago!) but it is not meaningful anymore. Python version specifiers are in the setup.cfg anyways :)
